README.md

```diff
@@ -13,52 +13,68 @@ in publications:
 
 * Olsson, Mats HM, Chresten R. Sondergaard, Michal Rostkowski, and Jan H. Jensen. "PROPKA3: consistent treatment of internal and surface residues in empirical pKa predictions." Journal of Chemical Theory and Computation 7, no. 2 (2011): 525-537. doi:[10.1021/ct100578z](https://doi.org/10.1021/ct100578z)
 
-See [propka.ki.ku.dk](http://propka.ki.ku.dk/) for the PROPKA web server,
-using the [tutorial](http://propka.ki.ku.dk/~luca/wiki/index.php/PROPKA_3.1_Tutorial).
+## Requirements
 
+PROPKA 3.2 requires Python 3.5 or higher.  Additional requirements are specified in the `requirements.txt` file and automatically satisfied when installing with [pip](https://pip.pypa.io).
 
 ## Installation
 
-Clone repository or unpack the tar ball and install with
-[setuptools](http://pythonhosted.org/setuptools/index.html) (note: if
-you don't have setuptools installed you will need an internet
-connection so that the installation procedure can download the
-required files):
+### PIP-based installation
 
-    cd propka-3.1
-    python setup.py install --user
+The easiest way to install PROPKA is via the [PyPI archive](https://pypi.org/project/PROPKA/) with the command
 
-This will install the `propka31` script in your executable directory,
-as configured for setuptools, for instance `~/.local/bin`. You can
-change the bin directory with the `--install-scripts` option. For
-example, in order to install in my `bin` directory in my home
-directory:
+    pip install propka
 
-    python setup.py install --user --install-scripts ~/bin
+As always, a virtual environment (e.g., via [virtualenv](https://pypi.org/project/virtualenv/)) is recommended when installing packages.
 
 For the purposes of testing or development, you may prefer to install PROPKA as
 an editable module via PIP by running
-```
-pip install -e .
-```
-from within a virtual environment (e.g., via [virtualenv](https://pypi.org/project/virtualenv/)).
 
-## Requirements
+    pip install -e .
+
```

Comment:
Move down to source install section
```suggestion

```

Reply:
Addressed in 9cd0da8ff3ff851ac0ab7c5bb428a1f8df328e58